pandas/core/indexing.py

```diff
@@ -1920,11 +1920,35 @@ def _convert_key(self, key, is_setter=False):
 
 
 class _iAtIndexer(_ScalarAccessIndexer):
-    """Fast integer location scalar accessor.
+    """
+    Selects a single value for a row/column pair by integer position.
 
-    Similarly to ``iloc``, ``iat`` provides **integer** based lookups.
-    You can also set using these indexers.
+    Useful if performance is a major concern and you only need to
+    get or set a value at a particular row/column.
 
+    See Also
+    --------
+    at : Selects a single value for a row/column label pair
```

Comment:
I think we will need to do `DataFrame.at` to make the automatic link work (and the same for the others below)